### PR TITLE
Updated and added files for 16.0 release description

### PR DESCRIPTION
**Pull request by gcomoretto**
After creating the 16.0 branch for the forthcoming 16.0 release, this are few changes in order to have the correct entries in the new branch.

### CODE REVIEW

File: releases/v16.rst [hunk 1]
```diff
@@ -0,0 +1,28 @@
+#######################
+LSST Stack 16.0 Release
+#######################
+
+Version 16.0 of the LSST Science Pipelines is now available. It is based on 2018 weekly 10.
+
+LSST Pipelines
+==============
+
+Full details of the LSST Science Pipelines 15.0 release can be found
```
Comment (jonathansick) [inline releases/v16.rst]: "LSST Science Pipelines 16.0"

File: releases/v16.rst [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+in the :ref:`extensive release notes <release-v16-0>`.  We have also
```
Comment (jonathansick) [inline releases/v16.rst]: Looks like you're using some weird hard-wrapping here. Make sure you use either one-sentence per line or one-paragraph per line line formatting for `pipelines_lsst_io`.

File: releases/v16.rst [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+published a `Characterization Metric Report <https://ls.st/DMTR-tbd>`_.
```
Comment (jonathansick) [inline releases/v16.rst]: You'll want to hold this PR until the characterization report is published so you make sure this URL is correct.

Comment (timj) [reply releases/v16.rst]: We can easily issue the docushare handle at any time.

Comment (gcomoretto) [reply releases/v16.rst]: I was considering to use tbd, in order to make clear that need to be updated before merging the 16.0 branch into the trunk. 
Simon told me that the characterization report should be ready this week.
This information should not be required now.

File: releases/v16.rst (hunk 2 above)
Comment (gcomoretto) [reply releases/v16.rst]: Here I let the original wrapping, I just updated from 15.0 to 16.0...

File: releases/v16.rst (hunk 1 above)
Comment (gcomoretto) [reply releases/v16.rst]: I missed this update. Al the rest should be fine:
- v16_0 in the toctree
- 16.0 defined as current (releases/notes.rst, I did not find other places to change it)

**Comment by jonathansick** [comment]
Nice, this build looks good.

**Pull request merged**